### DISCUSSION TITLE
Error npx prisma introspect to conect MYSQL Server 5

### DISCUSSION THREAD

**Discussion by Daniek3d** [prisma/prisma#2886] [Q&A] [answered]
Hello, when connect to Mysql Server 5 with npx prisma introspect to generate schemas it dont work,
In mysql server exist the database his y access in mysql workbench, the string connection y the same in both.

its my config files.

_datasource db {
provider = "mysql"
url = "mysql://root:manager@X.X.X.X:3306/his"
}

generator client {
provider = "prisma-client-js"
}_

this my debug error:

IntrospectionEngine:rpc {
IntrospectionEngine:rpc jsonrpc: '2.0',
IntrospectionEngine:rpc error: {
IntrospectionEngine:rpc code: 4466,
IntrospectionEngine:rpc message: 'An error happened. Check the data field for details.',
IntrospectionEngine:rpc data: {
IntrospectionEngine:rpc is_panic: false,
IntrospectionEngine:rpc message: "Error in connector: Error querying the database: Error querying the database: Error querying the database: Server error: ERROR HY000 (1193): Unknown system variable 'socket''", IntrospectionEngine:rpc backtrace: null IntrospectionEngine:rpc } IntrospectionEngine:rpc }, IntrospectionEngine:rpc id: 1 IntrospectionEngine:rpc } +352ms Error: Error: Error in connector: Error querying the database: Error querying the database: Error querying the database: Server error: ERROR HY000 (1193): Unknown system variable 'socket''

thanks.

**Comment by ryands17** [comment]
Hey @Daniek3d :wave: 
Is the version of MySQL 5.5?

**Comment by Daniek3d** [comment, marked as answer]
@ryands17  the version is 5.0.77

**Reply by ryands17** [reply]
@Daniek3d Unfortunately Prisma only supports MySQL 5.6 and above. Could you if possible update the version and try again?

**Comment by Daniek3d** [comment]
it is a legacy system and does not work with new versions of myql.
I will have to use another orm or access direct.
Thank you @ryands17 .

**Comment by ludralph** [comment]
Hi @Daniek3d,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
